sys/dev/athn/if_athn_usb.c

```diff
@@ -1207,17 +1332,18 @@ athn_if_bulk_tx_callback(struct usb_xfer *xfer, usb_error_t error)
 
 	usbd_xfer_status(xfer, &actlen, NULL, NULL, NULL);
 
-	printf("TTTT: BULK TX Xfer callback: error = %s, state = %s, actlen = %d, endpoint = 0x%lx\n",
-		   usbd_errstr(error), state_str, actlen, (long)xfer->endpoint);
+	// printf("TTTT: BULK TX Xfer callback: error = %s, state = %s, actlen = %d, endpoint = 0x%lx\n",
+	// 	   usbd_errstr(error), state_str, actlen, (long)xfer->endpoint);
 
 	switch(state) {
 	case USB_ST_TRANSFERRED:
 		data = STAILQ_FIRST(&usc->sc_tx_active[which]);
 		if (data == NULL)
 			goto tr_setup;
-		device_printf(sc->sc_dev, "%s: transfer done %p\n", __func__, data);
+		// device_printf(sc->sc_dev, "%s: transfer done %p\n", __func__, data);
 		STAILQ_REMOVE_HEAD(&usc->sc_tx_active[which], next);
 		athn_usb_txeof(xfer, data);
+		athn_freebuf(sc, data);
 		STAILQ_INSERT_TAIL(&usc->sc_tx_inactive, data, next);
```

Comment:
You didn't edit that, but I wonder where that line came from.

Reply:
I added it with raw_xmit.

Reply:
I mean the line below your change but nvm.